addons/website_blog/models/website_blog.py

```diff
@@ -257,6 +257,13 @@ def _notify_record_by_inbox(self, message, recipients_data, msg_vals=False, **kw
     def _default_website_meta(self):
         res = super(BlogPost, self)._default_website_meta()
         res['default_opengraph']['og:description'] = res['default_twitter']['twitter:description'] = self.subtitle
+        res['default_opengraph']['og:type'] = 'article'
+        res['default_opengraph']['article:published_time'] = self.post_date
+        res['default_opengraph']['article:modified_time'] = self.write_date
+        res['default_opengraph']['article:author'] = self.author_id.sudo().name
+        res['default_opengraph']['article:section'] = self.subtitle
```

Comment:
That's already the case with description (not related to this PR), so I guess empty og tag are not a problem? eg no `subtitle`.

Reply:
I don't think it matters, I couldn't find explicit info about it that having an empty `subtitle` having any issue. If you like I can wrap it in an if.

Reply:
what about smthing like:

```diff
diff --git addons/website/views/website_templates.xml addons/website/views/website_templates.xml
index 1d8ba8ef25e..5bdfc82be62 100644
--- addons/website/views/website_templates.xml
+++ addons/website/views/website_templates.xml
@@ -205,8 +205,15 @@
                 <!-- OpenGraph tags for Facebook sharing -->
                 <t t-set="opengraph_meta" t-value="website_meta.get('opengraph_meta')"/>
                 <t t-if="opengraph_meta">
-                    <t t-foreach="opengraph_meta" t-as="og_meta">
-                        <meta t-att-property="og_meta" t-att-content="opengraph_meta[og_meta]"/>
+                    <t t-foreach="opengraph_meta" t-as="property">
+                        <t t-if="isinstance(opengraph_meta[property], list)">
+                            <t t-foreach="opengraph_meta[property]" t-as="meta_content">
+                                <meta t-att-property="property" t-att-content="meta_content"/>
+                            </t>
+                        </t>
+                        <t t-else="">
+                            <meta t-att-property="property" t-att-content="opengraph_meta[property]"/>
+                        </t>
                     </t>
                 </t>
                 <!-- Twitter tags for sharing -->
```

Reply:
and 
article:section - string - A high-level section name. E.g. Technology

so section is not subtitle !!!
Just ignore this since it is not configurable in the user interface.

If you are an expert of blog and want to optimze your self your og, you can still make a custo using your own meta section that should not change according the blog post

Reply:
@JKE-be your view changes make no difference as in the backend (Python side) it is still a dict and a dict only takes unique keys, right? Or do I miss something?
As for the high-level section: I know it is not the same but there are more websites/plugins that do this to bump up the SEO, even though it is not a subtitle. We can either remove it or check if there is an h2 in the page and take that if you'd like.

Reply:
That's the tag entry that is a list, composed of differents tags.
If the opengraph element is a list (which it will be with tags) it goes through jke code
Else, act normal

Reply:
you miss something

res['default_opengraph']['article:tag'] = ','.join(self.tag_ids.mapped('name'))

1 key in dict, several values

Reply:
Ahh yes. Let me test your idea.

Reply:
Hmm, nope, it have to be different `article:tag` keys, mapping them with a comma will make the Facebook object debugger fail.
See https://stackoverflow.com/questions/36230298/how-to-add-multiple-tags-article-tag-in-facebook-open-graph

Reply:
res['default_opengraph']['article:tag'] = self.tag_ids.mapped('name')

Reply:
in thsi case, value is instance of 'list', so it will create a <meta/>  by tag in your list with the previous code from here: https://github.com/odoo/odoo/pull/35060#discussion_r306710390

Reply:
ho, thats a clever idea Jérémy. Will check all cases & update PR. I think this will be perfect. 👍